client/src/Movies/MovieCard.js

```diff
@@ -1,7 +1,30 @@
 import React from 'react';
 
+
+
+
 const MovieCard = props => {
-  return;
+  return (
+   
+      <div className="movie-card">
```

Comment:
Refactoring the code to only use this one component for multiple things is probably the best example to show how React components are re-usable.